torch_xla_py/__init__.py

```diff
@@ -0,0 +1,7 @@
+import warnings
```

Comment:
We cannot do this unfortunately.
We need to have per-file redirects, as otherwise someone using the old API with multiprocessing, will end up loading xla_model before the setup is ready, with potential bad results.
Although more files, I'd feel more confident in a per-module load, instead of a load-everything way.